packages/edit-post/src/components/layout/index.js

```diff
@@ -245,14 +243,16 @@ function Layout( { styles } ) {
 				notices={ <EditorSnackbars /> }
 				content={
 					<>
-						{ ! isDistractionFree && <EditorNotices /> }
+						{ ( ! isDistractionFree || ! isLargeViewport ) && (
+							<EditorNotices />
+						) }
```

Comment:
Does this fix another issue? Because it doesn't match `isDistractionFreeMode && isLargeViewport`.

Reply:
If the original condition is rewritten as is, it will be `! ( isDistractionFree && isLargeViewport )`.
Isn't this the same as `( ! isDistractionFree || ! isLargeViewport )`? 🤔 Personally, I think this writing style is easier to read.

Reply:
I think we only want to hide the notices in "distraction-free" mode. We can remove the `! isLargeViewport` part.

Reply:
Your're right, I have confirmed that removing that section will produce the same expected results as before.

In addition, I have made the following changes in 77ab658 to address the following issues:

> I noticed that the distraction-free mode could not be released unless the notice message bar was turned off, because it overlapped the header area. This may be a bug, but I think it should be addressed in a separate issue.

This should completely hide the notice message in distraction-free mode.

Reply:
I would prefer if we address those issues separately. Let's just fix the metabo issue in this PR.

Reply:
I would prefer if we address those issues separately. Let's just fix the metabo issue in this PR.